src/tensorlake/workflows/interface/__init__.py

```diff
@@ -0,0 +1,36 @@
+from .decorators import api, cls, function, reducer
+from .deploy import deploy
+from .exceptions import APIException, RequestException, RequestNotFinished
+from .function import Function
+from .function_call import FunctionCall
+from .package import Package, define_package
+from .request import Request
+from .request_context import (
+    RequestContext,
+    RequestContextPlaceholder,
+    RequestProgress,
+    RequestState,
+)
+from .run import local_run, remote_run
+
+__all__ = [
+    "api",
+    "cls",
+    "function",
+    "local_run",
```

Comment:
What do you think about this idea that just crossed my mind:

What if instead of having `local_run` and `remote_run`, we have one only `run` method with a parameter `remote=False`. That might give people easier control flow, see this example:

```python
request = tensorlake.run(func, payload_json="") // run this locally
request = tensorlake.run(func, payload_json="", remote=True) // run this remotely

import os
remote = True if os.getenv("TENSORLAKE_REMOTE") is not None
request = tensorlake.run(func, payload_json="", remote=remote)
```

/cc @diptanu

Reply:
I like this suggestion because it will require no code change to go from local to remote. I believe the current API was designed to make it easy for people to make remote and local runs more explicit.